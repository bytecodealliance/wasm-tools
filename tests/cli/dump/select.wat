;; FAIL: dump %
;; This fails because wasmprinter can't (yet) print an invalid multi-value select.
;; Can be changed back to a "RUN" test once that code lands.

(module
    (func
        select
        select (result)
        select (result i32)
        select (result i32 i32)
    )
)
