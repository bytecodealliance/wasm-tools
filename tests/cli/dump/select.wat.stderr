error: invalid result arity (at offset 0x18)
